Pages/Pag81751.NSPPerformanceTest.al

```diff
@@ -152,119 +195,177 @@ page 81751 "NSP Performance Test"
         ListOfText: List of [Text];
         StartDateTime: DateTime;
         i: Integer;
+        StartSqlStatements, StartSqlRowsRead : BigInteger;
     begin
+        DeleteEntries();
+        StartSqlStatements := SessionInformation.SqlStatementsExecuted;
+        StartSqlRowsRead := SessionInformation.SqlRowsRead;
         StartDateTime := CurrentDateTime();
         for i := 1 to NumberOfIterations do
-            ListOfText.Add(NoSeriesManagement.GetNextNo(NoSeriesCodeGapsLbl, Today(), true));
-        Message('Number series with gaps: %1 iterations took %2.', NumberOfIterations, CurrentDateTime() - StartDateTime);
+            //ListOfText.Add(NoSeriesManagement.GetNextNo(NoSeriesCodeGapsLbl, Today(), true));
+            CreateEntry3(NoSeriesManagement.GetNextNo(NoSeriesCodeGapsLbl, Today(), true));
+        Result.Log('TestNoSeriesGaps', CurrentDateTime() - StartDateTime, NumberOfIterations, comment, SessionInformation.SqlRowsRead - StartSqlRowsRead, SessionInformation.SqlStatementsExecuted - StartSqlStatements);
+        if ShowMessage then
+            Message('Number series with gaps: %1 iterations took %2.', NumberOfIterations, CurrentDateTime() - StartDateTime);
+        Commit;
     end;
 
-    local procedure TestNumberSequence()
+    local procedure TestNumberSequence(ShowMessage: Boolean)
     var
         ListOfText: List of [Text];
         StartDateTime: DateTime;
         i: Integer;
+        StartSqlStatements, StartSqlRowsRead : BigInteger;
     begin
+        DeleteEntries();
+        StartSqlStatements := SessionInformation.SqlStatementsExecuted;
+        StartSqlRowsRead := SessionInformation.SqlRowsRead;
         StartDateTime := CurrentDateTime();
         for i := 1 to NumberOfIterations do
-            ListOfText.Add(Format(NumberSequence.Next(NumberSequenceLbl)));
-        Message('Number sequence: %1 iterations took %2.', NumberOfIterations, CurrentDateTime() - StartDateTime);
+            //ListOfText.Add(Format(NumberSequence.Next(NumberSequenceLbl)));
+            CreateEntry3(Format(NumberSequence.Next(NumberSequenceLbl)));
+        Result.Log('TestNumberSequence', CurrentDateTime() - StartDateTime, NumberOfIterations, comment, SessionInformation.SqlRowsRead - StartSqlRowsRead, SessionInformation.SqlStatementsExecuted - StartSqlStatements);
+        if ShowMessage then
+            Message('Number sequence: %1 iterations took %2.', NumberOfIterations, CurrentDateTime() - StartDateTime);
+        Commit;
     end;
 
-    local procedure TestNumberIncrement()
-    var
-        ListOfText: List of [Text];
-        StartDateTime: DateTime;
-        i: Integer;
-    begin
-        StartDateTime := CurrentDateTime();
-        for i := 1 to NumberOfIterations do
-            ListOfText.Add(Format(i));
-        Message('Number increment: %1 iterations took %2.', NumberOfIterations, CurrentDateTime() - StartDateTime);
-    end;
+    // local procedure TestNumberIncrement(ShowMessage: Boolean)
```

Comment:
@NAVFreak 
In principle, it is better not to leave the code commented out.  Also, it is not quite clear why this test was commented out?